admin/app.js

```diff
@@ -48,7 +48,7 @@ azure.startInsightsIfConfigured()
 
 const unsetVars = []
 Object.keys(config).map((key) => {
-  if (!config[key] && !devWhitelist.includes(key)) {
+  if (config[key] === undefined && !devWhitelist.includes(key)) {
```

Comment:
why detect specifically for undefined?
The way it was done before it would also catch null and empty strings as well

Reply:
Consider:
```
AUTO_MARK=false yarn start
```
is a valid setting.  I am converting the string 'false' to the boolean value - `false` when the config is initialised - not all config values are strings (although all environment values are strings)

So we have to allow Falsey values in the config.

Reply:
I see

Reply:
So question is how do we detect null, empty strings then (if they do appear in the config)?

Reply:
If they have been defined by the user as that then it's accepted.